lightningd/opening/opening.c

```diff
@@ -209,7 +209,6 @@ static u8 *funder_channel(struct state *state,
 			  const struct utxo *utxos,
 			  const struct ext_key *bip32_base)
 {
-	const tal_t *tmpctx = tal_tmpctx(state);
```

Comment:
I don't understand why we are switching to allocating everything off of `state`. The `towire_open_channel` result and `towire_funding_created` seem like good candidates to continue to be allocated off of `tmpctx`, which would release the memory eagerly. Or am I missing the point here?